src/components/Sidebar.tsx

```diff
@@ -64,40 +64,43 @@ const Sidebar = ({ sidebarState }: { sidebarState: SidebarState }) => {
           <Link to="/">
             <Logo alt="alephium" src={theme.name === 'light' ? logoLight : logoDark} />
           </Link>
+          <NetworkMenu
+            label={networkType === 'mainnet' ? 'Mainnet' : 'Testnet'}
+            icon={networkType === 'mainnet' ? <NetworkLogo network="mainnet" /> : <NetworkLogo network="testnet" />}
+            items={[
+              {
+                text: 'Mainnet',
+                onClick: () => {
+                  window.location.assign('https://explorer.alephium.org')
+                },
+                icon: <NetworkLogo network="mainnet" />
+              },
+              {
+                text: 'Testnet',
+                onClick: () => {
+                  window.location.assign('https://testnet.alephium.org')
```

Comment:
I noticed that even if I am on the testnet network and click on the Testnet menu item, the page refreshes, which is unnecessary. I propose the following solution:

```js
  const switchToNetwork = (network: string) => {
    if (networkType !== network) {
      window.location.assign(
        networkType === 'mainnet' ? 'https://explorer.alephium.org' : `https://testnet.alephium.org`
      )
    }
  }
```

and in the render:

```js
        <NetworkMenu
          label={networkType === 'mainnet' ? 'Mainnet' : 'Testnet'}
          icon={networkType === 'mainnet' ? <AlephiumLogoMainnet /> : <AlephiumLogoTestnet />}
          items={[
            {
              text: 'Mainnet',
              onClick: () => switchToNetwork('mainnet'),
              icon: <AlephiumLogoMainnet />
            },
            {
              text: 'Testnet',
              onClick: () => switchToNetwork('testnet'),
              icon: <AlephiumLogoTestnet />
            }
          ]}
          direction={'up'}
        />
```

Reply:
You're not using `isMainnet` :) But yeah, will do.

Reply:
Done